Assignment2/arcecosta-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,70 @@
+{
+	"@context":{
+		"xsd":"http://www.w3.org/2001/XMLSchema#",
+		"s": "http://www.ontologies.org/ontologies/",
+		"prop": "http://www.ontologies.org/properties/",		
+		"includes": "prop:includes",
+		"hasMeasurement": "prop:measurements",
+		"hasOwner": "prop:hasOwner",
+		"hasName": "prop:hasName",
+		"hasTemperature": "prop:hasTemperature",
+		"atTime":"prop:atTime"
+	},
+
+		"Class01":{

```

Comment:
You don't need to define "Class01": because it isn't an attribute, it is a resource, thus I suggest you remove it. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes.